packages/perspective-jupyterlab/test/config/jupyter/jlab_start.js

```diff
@@ -18,7 +18,11 @@ const PACKAGE_ROOT = path.join(__dirname, "..", "..", "..");
  */
 const kill_jlab = () => {
     console.log("-- Cleaning up Jupyterlab process");
-    execute`ps aux | grep -i '[j]upyter-lab --no-browser' | awk '{print $2}' | xargs kill -9 && echo "[perspective-jupyterlab] JupyterLab process terminated"`;
+    try {
+        execute`ps aux | grep -i '[j]upyter-lab --no-browser' | awk '{print $2}' | xargs kill -9 && echo "[perspective-jupyterlab] JupyterLab process terminated"`;
+    } catch {
+        // ignore
+    }
```

Comment:
`execute()` does not throw (`execute_throw()` does, 'nat), this looks like debug code that snuck in